code/pilotfile/plr.cpp

```diff
@@ -139,6 +139,8 @@ void pilotfile::plr_read_hud()
 
 	HUD_config.rp_flags = cfread_int(cfp);
 	HUD_config.rp_dist = cfread_int(cfp);
+	if (HUD_config.rp_dist < 0 || HUD_config.rp_dist >= RR_MAX_RANGES)
+		Error(LOCATION, "Player file has invalid radar range %d\n", HUD_config.rp_dist);
```

Comment:
All the other failures in the plr parser are treated as non-fatal, so I'm not sure if this should, too.
Maybe a `Warning`?